app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductActivity.java

```diff
@@ -71,6 +80,51 @@ protected void onCreate(Bundle savedInstanceState) {
         if (!Utils.isHardwareCameraInstalled(this)) {
             mButtonScan.setVisibility(View.GONE);
         }
+
+        BottomNavigationView bottomNavigationView = (BottomNavigationView)findViewById(R.id.bottom_navigation);
+        BottomNavigationViewHelper.disableShiftMode(bottomNavigationView);
```

Comment:
I'm not sure this is the best idea, it makes the bottom navigation bar look _odd_

Reply:
@Karljoones thanks i will keep shifting mode